pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPoliciesImpl.java

```diff
@@ -178,7 +182,9 @@ public static OffloadPoliciesImpl create(String driver, String region, String bu
                                              String role, String roleSessionName,
                                              String credentialId, String credentialSecret,
                                              Integer maxBlockSizeInBytes, Integer readBufferSizeInBytes,
-                                             Long offloadThresholdInBytes, Long offloadDeletionLagInMillis,
+                                             Long offloadThresholdInBytes,
+                                             //Long offloadTimeThreshold, //TODO
```

Comment:
Is It still useful?

Reply:
It's for the CLI and RestAPI, I'll remove here